content/pt/docs/reference/access-authn-authz/authentication.md

```diff
@@ -0,0 +1,892 @@
+---
+reviewers:
+- femrtnz
+- jcjesus
+title: Autenticação
+content_type: concept
+weight: 10
+---
+ 
+<!-- overview -->
+Essa página demonstra uma visão geral sobre autenticação
+ 
+<!-- body -->
+## Usuários em Kubernetes
+ 
+Todos os clusters Kubernetes possuem duas categorias de usuários: contas de serviço gerenciadas por Kubernetes e usuários normais.
+ 
+Assume-se que um serviço independente do cluster gerencia usuários normais das seguintes formas:
+ 
+- Um administrador distribuindo chaves privadas
+- Uma base de usuários como Keystone ou Google Accounts
+- Um arquivo com uma lista de nomes de usuários e senhas
+ 
+Neste quesito, _Kubernetes não possui objetos que possam representar as contas de um usuário normal._ Usuários normais não podem ser adicionados ao _cluster_ através de uma chamada para a API.
+ 
+Apesar de um usuário normal não poder ser adicionado através de uma chamada para a API, qualquer usuário que apresente um certificado válido e assinado pela autoridade de certificados (CA) do _cluster_ é considerado autenticado. Nesta configuração, Kubernetes determina o nome do usuário baseado no campo de nome comum no sujeito (_subject_) do certificado (por exemplo: "/CN=bob"). A partir daí, o subsistema de controle de acesso baseado em função (RBAC) determina se o usuário é autorizado a realizar uma operação específica sobre o recurso. Para mais detalhes, veja a referência sobre o tópico de usuários normais dentro de requisição de certificado (/docs/reference/access-authn-authz/certificate-signing-requests/#normal-user).
```

Comment:
Just a small fix:
`requisição de certificado (/docs/reference/access-authn-authz/certificate-signing-requests/#normal-user)` is
`[requisição de certificado](/docs/reference/access-authn-authz/certificate-signing-requests/#normal-user)`

Reply:
Fixed on commit 9b5815e